python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -1854,8 +1855,27 @@ def __init__(self, schema: Schema, name: str, options: Any, df: IR):
         elif self.name == "row_index":
             col_name, offset = options
             self.options = (col_name, offset)
+        elif self.name == "fast_count":
+            _, scan_type, _ = options
+            typ = scan_type[0]
+            if typ != "parquet":
```

Comment:
Note: We cannot get the row count for csv and json sources like we do parquet because we dont have the row count available to as metadata. How should we get the row count in those cases?

One idea would be to count the number of `\n` characters.